feature/home/src/main/java/com/epmedu/animeal/home/data/model/FeedSpot.kt

```diff
@@ -0,0 +1,10 @@
+package com.epmedu.animeal.home.data.model
+
+data class FeedSpot(
```

Comment:
Why data class should have values by default?

Reply:
Agree. Looks cleaner without initialisations

Reply:
Updated

Reply:
Applied workaround by setting `Modifier.defaultMinSize(minHeight = 1.dp)` around sheet content